src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -121,13 +136,13 @@ public CertificateManager getCertificateManager() {
     }
 
     @SuppressWarnings("PMD.UnusedFormalParameter")
-    private void handleConfigurationChange(WhatHappened whatHappened, Node childNode) {
+    private void handleConfigurationChange(WhatHappened whatHappened, Topics deviceGroupsTopics) {
```

Comment:
nit: method could use a more specific name, might want it to match w/ `updateCAType`:  `updateDeviceGroups`

Reply:
Fixed